hack/jenkins/job-configs/kubernetes-e2e.yaml

```diff
@@ -358,13 +358,10 @@
             test-owner: 'beeps'
             provider-env: '{gke-provider-env}'
             job-env: |
-                # XXX Not a unique project
                 export E2E_NAME="e2e-gke-ingress"
```

Comment:
Why not just be explicit? it's just easier to figure out the options by looking at the config, even if you don't understand the defaults.